python/lsst/ctrl/oods/cacheCleaner.py

```diff
@@ -139,14 +140,15 @@ async def getFilesOlderThan(self, seconds, directory):
         """
         files = []
 
-        for dirName, subdirs, fileList in os.walk(directory):
-            for fname in fileList:
-                await asyncio.sleep(0)
-                fullName = os.path.join(dirName, fname)
-                stat_info = os.stat(fullName)
-                modification_time = stat_info.st_mtime
-                if modification_time < seconds:
-                    files.append(fullName)
+        scanner = Scanner()
+        async for entry in scanner.scan(directory):
+            if entry.is_dir():
+                continue
+            fullName = entry.path
```

Comment:
Can we rename `fullName` to `full_name`?

Reply:
Renamed.